install_config/configuring_extended_ldap_attributes.adoc

```diff
@@ -0,0 +1,175 @@
+= Configuring Extended LDAP Attributes
+{product-author}
```

Comment:
-----BEGIN PGP SIGNED MESSAGE-----
Hash: SHA1

For the record, I hereby grant Red Hat, Inc. permission in writing to reuse content from the following blog posts and license this content as necessary for the purpose of producing documentation.
 * https://sgallagh.wordpress.com/2016/05/26/openshift-and-sssd-part-1-basic-ldap-authentication/
 * https://sgallagh.wordpress.com/2016/05/27/openshift-and-sssd-part-2-ldap-form-authentication/
 * https://sgallagh.wordpress.com/2016/05/27/openshift-and-sssd-part-3-extended-ldap-attributes/
-----BEGIN PGP SIGNATURE-----
Version: GnuPG v2

iEYEARECAAYFAldgHb4ACgkQeiVVYja6o6NMMgCgmrGlg++ZDOIuZf8xPmnT6jDe
XvgAn3xz0zbNPiRwf44VRwGKWwyO4WPK
=xdiB
-----END PGP SIGNATURE-----